get-started/part5.md

```diff
@@ -85,7 +85,7 @@ with the following. Be sure to replace `username/repo:tag` with your image detai
       visualizer:
         image: dockersamples/visualizer:stable
         ports:
-          - "8080:8080"
+          - "8080:8080":
```

Comment:
Is this colon intentional?

Reply:
I can't speak for whether it is intentional but I can confirm that it doesn't actually break YAML.

Reply:
it's not intentional, I believe. I'll remove it. Working with @shin-  on this problem today.